Source/WebCore/crypto/cocoa/CryptoAlgorithmAESGCMMac.cpp

```diff
@@ -81,10 +86,15 @@
     return WTFMove(plainText);
 }
 
+#if HAVE(SWIFT_CPP_INTEROP)
```

Comment:
I think it would be a lot nicer for our { and } matching to have this #if inside the function, then just have an #else UNUSED_PARAM(useCryptoKit).  Same everywhere.

Reply:
done